### PR TITLE
AAP-20823

### PR DESCRIPTION
**Pull request by g-murray**
Performing an editorial review of the [Red Hat Ansible Automation Platform Performance Considerations for Operator Based Installations](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_performance_considerations_for_operator_based_installations/index) doc.

Location: downstream > Titles > ocp_performance_guide

### CODE REVIEW

File: downstream/modules/platform/con-pod-specification-mods.adoc [hunk 1]
```diff
@@ -8,13 +8,13 @@ Pods are the equivalent of a machine instance (physical or virtual) to a contain
 Each pod is allocated its own internal IP address, therefore owning its entire port space, and containers within pods can share their local storage and networking.
 
 Pods have a life cycle.
-They are defined, then they are assigned to run on a node, then they run until their container(s) exit or they are removed for some other reason.
-Pods, depending on policy and exit code, may be removed after exiting, or may be retained to enable access to the logs of their containers.
+They are defined, then they are assigned to run on a node, then they run until their containers exit or they are removed for some other reason.
+Pods, depending on policy and exit code, can be removed after exiting, or can be retained to enable access to the logs of their containers.
 
-{PlatformName} provides a simple default Pod specification, however, you can provide a custom YAML, or JSON document that overrides the default Pod specification.
+{PlatformName} provides a simple default Pod specification, however, you can provide a custom YAML, or JSON document that overrides the default pod specification.
```
Comment (michellemacrh) [inline downstream/modules/platform/con-pod-specification-mods.adoc:14]: I can't remember the exact rule for pods/Pods but "simple default Pod specification" is uppercase, I presume this is OK?

File: downstream/modules/platform/con-pod-specification-mods.adoc [hunk 2]
```diff
@@ -83,15 +83,15 @@ A pod definition can specify the basis of a name with the `generateName` attribu
...
+The default context is very restrictive but administrators can change this as required.
 | `volumeMounts:` | The container specifies where external storage volumes should be mounted within the container.
 In this case, there is a volume for storing the registry’s data, and one for access to credentials the registry needs for making requests against the OpenShift Online API.
 | `ImagePullSecrets` | A pod can contain one or more containers, which must be pulled from some registry.
-If containers come from registries that require authentication, you can provide a list of `ImagePullSecrets` that refer to `ImagePullSecrets` present in the namespace.
+If containers come from registries that require authentication, you can give a list of `ImagePullSecrets:` that refer to `ImagePullSecrets` present in the namespace.
 Having these specified enables {OCP} to authenticate with the container registry when pulling the image.
 For further information, see link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[Managing resource containers] in the kubernetes documentation.
```
Comment (michellemacrh) [inline downstream/modules/platform/con-pod-specification-mods.adoc:96]: Should Kubernetes be capitalized in "in the kubernetes documentation"?

**Review by michellemacrh — COMMENTED** [review]
2 small suggestions 👍

File: downstream/modules/platform/con-pod-specification-mods.adoc (hunk 2 above)
Comment (g-murray) [reply downstream/modules/platform/con-pod-specification-mods.adoc:96]: You are correct, I've fixed it now :)

**Comment by michellemacrh** [comment]
LGTM

**Pull request merged**